.github/workflows/test.yml

```
@@ -12,7 +12,7 @@ env:
 
 jobs:
   rust:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Given that we technically still support ubuntu 14, I'd probably just update this to 20.04.  Also, random breakage when the meaning of `latest` changes isn't great.
```suggestion
    runs-on: ubuntu-20.04
```

---

Sure, we just seem to use latest everywhere else... so that's what I went with.

```
# rg --hidden -IN runs-on  | sort | uniq -c
     10     runs-on: ${{ matrix.os }}
      1     runs-on: ${{ matrix.platform }}
      1     runs-on: macos-10.15
      3     runs-on: macos-latest
     10     runs-on: ubuntu-18.04
     27     runs-on: ubuntu-20.04
      4     runs-on: ubuntu-22.04
      2         runs-on: ubuntu-latest
      1      runs-on: ubuntu-latest
    162     runs-on: ubuntu-latest
      1   #   runs-on: ubuntu-latest
      2     runs-on: windows-2019
      4     runs-on: windows-latest
```

---

Not all the same use cases.  If they're not compiling C/C++ code then it doesn't really matter, but in this case it might.